daemon/container_operations_solaris.go

```diff
@@ -45,5 +107,15 @@ func detachMounted(path string) error {
 }
 
 func isLinkable(child *container.Container) bool {
+	// A container is linkable only if it belongs to the default network
+	_, ok := child.NetworkSettings.Networks["bridge"]
```

Comment:
`_, ok := child.NetworkSettings.Networks[runconfig.DefaultDaemonNetworkMode().NetworkName()]` feels safer (and is what's use in `container_operations_unix.go`).